eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -406,6 +409,7 @@ applySubst subst@Subst{..} = \case
   obj@(MetaObject x) -> fromMaybe obj $ lookup x objectMetas
   Termination -> Termination
   MetaSubstThis obj thisObj -> MetaSubstThis (applySubst subst thisObj) (applySubst subst obj)
+  MetaContextualize obj thisObj -> MetaContextualize (applySubst subst thisObj) (applySubst subst obj)
```

Comment:
Why are `obj` and `thisObj` swapped here?

Reply:
Good catch. I don't know why. I've changed the order here [c4efd52](https://github.com/objectionary/eo-phi-normalizer/pull/523/commits/c4efd527df948250c42245efe6e8408e61b2f7af) for both substitution and contextualization as the tests pass, and this swapped order does not seem to be intentional.

@Anatolay does this influence anything?